cannot compile on FreeBSD

When building crate on FreeBSD i got:

```
warning: src/file_lock.c:12:17: error: use of undeclared identifier 'SEEK_SET'
warning:   fl.l_whence = SEEK_SET;
warning:                 ^
warning: src/file_lock.c:31:17: error: use of undeclared identifier 'SEEK_SET'
warning:   fl.l_whence = SEEK_SET;
warning:                 ^
warning: 2 errors generated.
error: failed to run custom build command for `file-lock v2.1.9`
```

--- a/src/file_lock.c
+++ b/src/file_lock.c
@@ -1,5 +1,6 @@
 #include <errno.h>
 #include <fcntl.h>
+#include <stdio.h>
 
 int c_lock(int fd, int is_blocking, int is_writable) {
   if (fd < 0) {

